CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# CONTRIBUTING
+
+Contributions to the XRPL-Standards repo are free, open source, and follow the following document publishing process.
+
+Any XRPL-Standards document can be referred to interchangeably as an "XLS" (XRPL Standard), "RFC" (Request for Comments), or "document".
+
+## Summary
+
+Copyright on all content is subject to the terms of the [MIT License](LICENSE).
+
+All contributors grant everyone:
+
+Copyright – a royalty-free license to use the copyrights for their contributions.
+Patent – a commitment to license on a royalty-free basis their essential patent claims reading on their contributions.
+
+## Background
+
+The work of the XRP Ledger community is open source, collaborative, and welcoming of all contributors participating in good faith. [Learn more about the XRP Ledger at XRPL.org](https://xrpl.org/).
+
+This XRPL-Standards process is influenced by the well-established [IETF RFC process](https://www.ietf.org/standards/process/informal/).
+
+## Process
+
+The XRPL-Standards process attempts to be easy to use, but also rigorous enough that there are permalinks to revisions of documents for reference purposes.
+
+### Gathering Feedback Before Submitting
+
+Please gather community input before opening a PR. Collecting such feedback helps to refine your concept. This step is required.
+
+Start a [Discussion](https://github.com/XRPLF/XRPL-Standards/discussions) under this repo.
+
+Discussions are suitable for early work-in-progress: ask, suggest, add, and make sweeping changes.
+
+Once settled, the discussion should be converted to a **PULL REQUEST** adding the standard to the repo with one or more commits. At this time, the discussion should be archived and closed from further comments, with any further comments being posted on the PR for fine-tuning and alignment with implementation (as appropriate).
```

Comment:
I suspect we might want to refine some of this guideline over time as "settled" may be hard to gauge sometimes